app/main.go

```diff
@@ -32,7 +32,7 @@ import (
 
 	"github.com/dustin/go-humanize"
 	"google.golang.org/appengine"
-	"google.golang.org/appengine/log"
+	log "github.com/sirupsen/logrus"
```

Comment:
Is there a specific reason why logrus was choosen?

E.g. https://github.com/go-logr/logr is an abstract interface that supports multiple implementations including logrus.

Reply:
Sounds good, let's use logrus directly at this stage.